test/tune/test_lexiflow.py

```diff
@@ -32,7 +32,7 @@ def _BraninCurrin(config):
     return {"brain": brain_result, "currin": currin_result}
 
 
-def test_lexiflow():
+def test_lexiflow(mode="absolute"):
```

Comment:
Add a test for (mode="percentage"). pytest will look for test function and won't execute 194-196.